app/controllers/admin/users_controller.rb

```diff
@@ -13,13 +13,13 @@ def new
     end
 
     def create
-      @user = User.new(user_params)
-      @user.confirmed_at = Time.now
+      @user = User.new(user_params.merge(confirmed_at: Time.zone.now))
```

Comment:
merge the password in this line as well. Also, create a constant for a password.